virt/virt-learn-more-about-openshift-virtualization.adoc

```diff
@@ -1,109 +1,73 @@
-
-| xref:../virt/about-virt.adoc#about-virt[Learn about {VirtProductName}]
-| xref:../virt/install/virt-enabling-virtctl.adoc#virt-enabling-virtctl[Enabling the `virtctl` client]
-| Viewing xref:../virt/logging_events_monitoring/virt-logs.adoc#virt-logs[logs] and xref:../virt/logging_events_monitoring/virt-events.adoc#virt-events[events]
-| xref:../virt/logging_events_monitoring/virt-collecting-virt-data.adoc#virt-collecting-virt-data[Getting Support]
-
-|
-| xref:../virt/virtual_machines/virt-create-vms.adoc#virt-create-vms[Creating virtual machines]
-| xref:../virt/logging_events_monitoring/virt-monitoring-vm-health.adoc#virt-monitoring-vm-health[Monitoring virtual machine health]
-|
-
-|
-| xref:../virt/virtual_machines/virt-manage-vmis.adoc#virt-manage-vmis[Managing virtual machines instances]
-| xref:../virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc#virt-managing-vm-snapshots[Creating and managing virtual machine snapshots]
-|
-
-|
-| xref:../virt/virtual_machines/virt-controlling-vm-states.adoc#virt-controlling-vm-states[Controlling virtual machine states]
-|
-|
-
-|
-| xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-accessing-vm-consoles[Accessing the virtual machine consoles]
-|
-|
-
-|
-| xref:../virt/virtual_machines/virt-managing-configmaps-secrets-service-accounts.adoc#virt-managing-configmaps-secrets-service-accounts[Pass configuration data to virtual machines using secrets, configuration maps, and service accounts]
-|
-|
-
-|
-|===
+You can install and configure a basic {VirtProductName} environment in order to explore its features and functionality.
+
+[NOTE]
+====
+Cluster configuration procedures require `cluster-admin` privileges.
+====
+
+[id="virt-getting-started-before-you-begin"]
+== Before you begin
+
+* Review xref:../virt/about-virt.adoc#about-virt[About {VirtProductName}].
+* Review the xref:../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[installation requirements].
+* Review the xref:../virt/virtual_machines/virtual_disks/virt-features-for-storage.adoc#virt-features-for-storage[storage types] required for cloning, snapshots, and live migration. See xref:../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[Using a CSI-enabled storage provider] for details.
+* Install the xref:../virt/install/installing-virt-web.adoc#installing-virt-web[Install the {VirtProductName} Operator] and the xref:../virt/install/virt-enabling-virtctl.adoc#virt-enabling-virtctl[`virtctl` tool].
+
+[id="virt-getting-started"]
+== Getting started
+
+Create a virtual machine:
+
+* Create xref:../virt/virtual_machines/virt-create-vms.adoc#virt-creating-vm-wizard-web_virt-create-vms[Red Hat Enterprise Linux virtual machines] using the wizard.
+* Create a Microsoft Windows virtual machine
+** xref:../virt/virtual_machines/virt-automating-windows-sysprep.adoc#virt-automating-windows-sysprep[Create and upload a Microsoft Windows boot source].
```

Comment:
Hmm. To be honest, we do not have good instructions for creating a Windows boot source. Maybe we should link to the [creating boot source](http://file.rdu.redhat.com/bgaydos/CNV-16812/virt/vm_templates/virt-creating-vm-template.html#virt-adding-a-boot-source-web_virt-creating-vm-template) section instead. I think this is a question for an SME.

Reply:
The Windows process is definitely not the same as the one in the main doc for Linux. We've always referred to the blog post. I've asked Chandler to confirm and instead pasted the two links from our boot source topics to the blogs. As I've said before, I don't know when the Windows process will become more similar to the Linux process, but it's still not there yet.

I'm guessing we have some sort of trademark catch-all that equates Windows to MS Windows, so I'll change it. I did ask Ingrid about it to confirm, still waiting for a ping back.